authors/apps/authentication/models.py

```diff
@@ -117,4 +115,16 @@ def get_short_name(self):
         """
         return self.username
 
-
+    @property
+    def token(self):
+        """
+        This method generates and returns a string of the token generated.
+        """
+        date = datetime.now() + timedelta(hours=24)
+
+        payload = {
+            'email': self.email,
+            'exp': int(date.strftime('%s')),
+        }
+        token = jwt.encode(payload, settings.SECRET_KEY, algorithm='HS256')
+        return token.decode()
```

Comment:
This feels like we are encoding the token... and then decoding it again. What's happening here?

Reply:
This is where we generate the token.
Supposed to be `return token`. Thanks for noting that.